src/strategies/convex/strategy-convex-stecrv.sol

```diff
@@ -0,0 +1,246 @@
+        uint256 claimableCvx = this.getHarvestableCvx();
+
+        uint256 crvEth = _estimateSell(univ2Router2, address(crv), claimableCrv);
+        uint256 ldoEth = _estimateSell(univ2Router2, address(ldo), claimableLdo);
+        uint256 cvxEth = _estimateSell(sushiRouter,  address(cvx), claimableCvx);
+
+        return crvEth.add(ldoEth).add(cvxEth);
+    }
+
+    function _estimateSell(address dex, address currency, uint256 amount) internal view returns (uint256 outAmount){
+        address[] memory path = new address[](2);
+        path[0] = currency;
+        path[1] = weth;
+        uint256[] memory amounts = UniswapRouterV2(dex).getAmountsOut(amount, path);
+        outAmount = amounts[amounts.length - 1];
+
+        return outAmount;
+    }
+
+    // **** Setters ****
+
+    function setKeepCVX(uint256 _keepCVX) external {
+        require(msg.sender == governance, "!governance");
+        keepCVX = _keepCVX;
+    }
+
+    // **** State Mutations ****
+
+    function deposit() public override {
+        uint256 _want = IERC20(want).balanceOf(address(this));
+        if (_want > 0) {
+            depositor.deposit(pid, _want, true);
+        }
+    }
+
+    function _withdrawSome(uint256 _amount)
+        internal
+        override
+        returns (uint256)
+    {
+        rewardPool.withdrawAndUnwrap(_amount, false);
+        return _amount;
+    }
+
+    function harvest() public override onlyBenevolent {
+        // Anyone can harvest it at any given time.
+        // I understand the possibility of being frontrun / sandwiched
+        // But ETH is a dark forest, and I wanna see how this plays out
+        // i.e. will be be heavily frontrunned/sandwiched?
+        //      if so, a new strategy will be deployed.
+
+        rewardPool.getReward();
+
+        uint256 _ldo = ldo.balanceOf(address(this));
+        uint256 _crv = crv.balanceOf(address(this));
+        uint256 _cvx = cvx.balanceOf(address(this));
+
+        if (_cvx > 0) {
+            // How much CVX to keep to restake?
+            uint256 _keepCVX = _cvx.mul(keepCVX).div(keepCVXMax);
+            if (_keepCVX > 0) {
+                IERC20(cvx).safeTransfer(
+                    IController(controller).treasury(),
+                    _keepCVX
+                );
+            }
+
+            // How much CVX to swap?
+            _cvx = _cvx.sub(_keepCVX);
+            _swapSushiswap(address(cvx), weth, _cvx);
+        }
+        if (_ldo > 0) {
+            _swapUniswap(address(ldo), weth, _ldo);
+        }
+        if (_crv > 0) {
+            _swapUniswap(address(crv), weth, _crv);
+        }
+        WETH(weth).withdraw(WETH(weth).balanceOf(address(this)));
+
+        uint256 _eth = address(this).balance;
+        stEth.submit{value: _eth/2}(strategist);
+        _eth = address(this).balance;
+        uint256 _stEth = stEth.balanceOf(address(this));
+
+        uint256[2] memory liquidity;
+        liquidity[0] = _eth;
+        liquidity[1] = _stEth;
```

Comment:
just use like this. uint256[2] memory liquidity = [_eth, _stEth]

Reply:
Combined liquidity declaration with initialization